Dev mode integration tests intermittent failures

Intermittent failures occurring in dev mode integration tests. Anywhere from 2 - 6 tests failing through GitHub actions. 
Dev mode is running with generateFeatures. 

Failures:
- ExplodedLooseAppWarTest currently consistently fails and need to be re-enabled.
- DevTest.testDirectoryTest
```
2022-01-20T17:02:51.4226247Z [INFO] Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 72.551 sec <<< FAILURE! - in net.wasdev.wlp.test.dev.it.DevTest
2022-01-20T17:02:51.4230223Z [INFO] testDirectoryTest(net.wasdev.wlp.test.dev.it.DevTest)  Time elapsed: 6.222 sec  <<< FAILURE!
2022-01-20T17:02:51.4230693Z [INFO] java.lang.AssertionError: 
2022-01-20T17:02:51.4234164Z [INFO] Last 100 lines of log at /home/runner/work/ci.maven/ci.maven/liberty-maven-plugin/target/it/dev-it/target/../resources/basic-dev-project/logFile.txt:
2022-01-20T17:02:51.4237940Z [INFO] ===================== START =======================
2022-01-20T17:02:51.4238554Z [INFO] [INFO] Server defaultServer created.
2022-01-20T17:02:51.4241533Z [INFO] [INFO] CWWKM2129I: Server defaultServer has been created at /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer.
2022-01-20T17:02:51.4256372Z [INFO] [INFO] Copying 2 files to /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer
2022-01-20T17:02:51.4256955Z [INFO] [INFO] CWWKM2144I: Update server configuration file server.xml from /tmp/temp3087577316882393089/src/main/liberty/config/server.xml.
2022-01-20T17:02:51.4257471Z [INFO] [INFO] CWWKM2144I: Update server configuration file bootstrap.properties from inlined configuration.
2022-01-20T17:02:51.4258299Z [INFO] [INFO] Running liberty:install-feature
2022-01-20T17:02:51.4258863Z [INFO] [INFO] CWWKM2102I: Using artifact based assembly archive : io.openliberty:openliberty-kernel:null:21.0.0.9:zip.
2022-01-20T17:02:51.4259382Z [INFO] [INFO] CWWKM2102I: Using installDirectory : /tmp/temp3087577316882393089/target/liberty/wlp.
2022-01-20T17:02:51.4259771Z [INFO] [INFO] CWWKM2102I: Using serverName : defaultServer.
2022-01-20T17:02:51.4260189Z [INFO] [INFO] CWWKM2102I: Using serverDirectory : /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer.
2022-01-20T17:02:51.4260706Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/server.xml for features and includes.
2022-01-20T17:02:51.4262053Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml for features and includes.
2022-01-20T17:02:51.4263423Z [INFO] [INFO] Installing features: [servlet-4.0, mpRestClient-1.2, cdi-2.0, jsonp-1.1, mpConfig-1.3, jaxrs-2.1]
2022-01-20T17:02:51.4263798Z [INFO] 
2022-01-20T17:02:51.4264521Z [INFO] [INFO] Product validation completed successfully.
2022-01-20T17:02:51.4268106Z [INFO] [INFO] The following features have been installed: jndi-1.0 jaxrsClient-2.1 mpRestClient-2.0 mpRestClient-1.2 mpConfig-1.3 jsonp-1.1 mpConfig-2.0 cdi-2.0 servlet-4.0 jaxrs-2.1 
2022-01-20T17:02:51.4272688Z [INFO] [INFO] Running liberty:deploy
2022-01-20T17:02:51.4274643Z [INFO] [INFO] CWWKM2102I: Using artifact based assembly archive : io.openliberty:openliberty-kernel:null:21.0.0.9:zip.
2022-01-20T17:02:51.4278479Z [INFO] [INFO] CWWKM2102I: Using installDirectory : /tmp/temp3087577316882393089/target/liberty/wlp.
2022-01-20T17:02:51.4304153Z [INFO] [INFO] CWWKM2102I: Using serverName : defaultServer.
2022-01-20T17:02:51.4305967Z [INFO] [INFO] CWWKM2102I: Using serverDirectory : /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer.
2022-01-20T17:02:51.4307717Z [INFO] [INFO] Copying 2 files to /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer
2022-01-20T17:02:51.4312631Z [INFO] [INFO] CWWKM2144I: Update server configuration file server.xml from /tmp/temp3087577316882393089/src/main/liberty/config/server.xml.
2022-01-20T17:02:51.4321415Z [INFO] [INFO] CWWKM2144I: Update server configuration file bootstrap.properties from inlined configuration.
2022-01-20T17:02:51.4322308Z [INFO] [INFO] CWWKM2185I: The liberty-maven-plugin configuration parameter "appsDirectory" value defaults to "dropins".
2022-01-20T17:02:51.4323324Z [INFO] [INFO] CWWKM2160I: Installing application dev-sample-proj-1.0-SNAPSHOT.war.xml.
2022-01-20T17:02:51.4325344Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/server.xml for features and includes.
2022-01-20T17:02:51.4328042Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml for features and includes.
2022-01-20T17:02:51.4328678Z [INFO] [INFO] Copying 2 files to /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer
2022-01-20T17:02:51.4330757Z [INFO] [INFO] CWWKM2144I: Update server configuration file server.xml from /tmp/temp3087577316882393089/src/main/liberty/config/server.xml.
2022-01-20T17:02:51.4331668Z [INFO] [INFO] CWWKM2144I: Update server configuration file bootstrap.properties from inlined configuration.
2022-01-20T17:02:51.4333777Z [INFO] [INFO] CWWKM2001I: Invoke command is [/tmp/temp3087577316882393089/target/liberty/wlp/bin/server, debug, defaultServer].
2022-01-20T17:02:51.4334238Z [INFO] [INFO] 
2022-01-20T17:02:51.4334849Z [INFO] [INFO] Listening for transport dt_socket at address: 7777
2022-01-20T17:02:51.4337579Z [INFO] [INFO] Launching defaultServer (Open Liberty 21.0.0.9/wlp-1.0.56.cl210920210824-2341) on OpenJDK 64-Bit Server VM, version 11.0.13+8 (en)
2022-01-20T17:02:51.4338423Z [INFO] [INFO] [AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
2022-01-20T17:02:51.4342140Z [INFO] [INFO] CWWKM2010I: Searching for CWWKF0011I: in /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/logs/messages.log. This search will timeout after 90 seconds.
2022-01-20T17:02:51.4346048Z [INFO] [INFO] [AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml
2022-01-20T17:02:51.4346745Z [INFO] [INFO] [AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
2022-01-20T17:02:51.4349691Z [INFO] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://fv-az32-87.xh5foii4tneu1oj5xw13sxakeb.cx.internal.cloudapp.net:9080/dev-sample-proj-1.0-SNAPSHOT/
2022-01-20T17:02:51.4350578Z [INFO] [INFO] [AUDIT   ] CWWKZ0001I: Application dev-sample-proj-1.0-SNAPSHOT started in 0.974 seconds.
2022-01-20T17:02:51.4352500Z [INFO] [INFO] [AUDIT   ] CWWKF0012I: The server installed the following features: [jaxrs-2.1, jaxrsClient-2.1, jsonp-1.1, servlet-4.0].
2022-01-20T17:02:51.4354316Z [INFO] [INFO] [AUDIT   ] CWWKF0011I: The defaultServer server is ready to run a smarter planet. The defaultServer server started in 4.333 seconds.
2022-01-20T17:02:51.4364803Z [INFO] [INFO] CWWKM2015I: Match number: 1 is {"type":"liberty_message","host":"fv-az32-87.xh5foii4tneu1oj5xw13sxakeb.cx.internal.cloudapp.net","ibm_userDir":"\/tmp\/temp3087577316882393089\/target\/liberty\/wlp\/usr\/","ibm_serverName":"defaultServer","message":"CWWKF0011I: The defaultServer server is ready to run a smarter planet. The defaultServer server started in 4.333 seconds.","ibm_threadId":"0000001f","ibm_datetime":"2022-01-20T17:01:57.463+0000","ibm_messageId":"CWWKF0011I","module":"com.ibm.ws.kernel.feature.internal.FeatureManager","loglevel":"AUDIT","ibm_sequence":"1642698117463_0000000000013"}.
2022-01-20T17:02:51.4365903Z [INFO] [INFO] ************************************************************************
2022-01-20T17:02:51.4396882Z [INFO] [INFO] *    Liberty is running in dev mode.
2022-01-20T17:02:51.4397402Z [INFO] [INFO] *        Automatic generation of features: [ On ]
2022-01-20T17:02:51.4398284Z [INFO] [INFO] *        To see the help menu for available actions, type 'h' and press Enter.
2022-01-20T17:02:51.4398935Z [INFO] [INFO] *        To stop the server and quit dev mode, press Ctrl-C or type 'q' and press Enter.
2022-01-20T17:02:51.4399550Z [INFO] [INFO] *        
2022-01-20T17:02:51.4399960Z [INFO] [INFO] *    Liberty server port information:
2022-01-20T17:02:51.4400634Z [INFO] [INFO] *        Liberty debug port: [ 7777 ]
2022-01-20T17:02:51.4401073Z [INFO] [INFO] ************************************************************************
2022-01-20T17:02:51.4401932Z [INFO] [INFO] Source compilation had errors.
2022-01-20T17:02:51.4402464Z [INFO] [INFO] Running liberty:generate-features
2022-01-20T17:02:51.4403194Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/src/main/liberty/config/server.xml for features and includes.
2022-01-20T17:02:51.4404008Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/src/main/liberty/config/configDropins/overrides/generated-features.xml for features and includes.
2022-01-20T17:02:51.4404843Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/src/main/liberty/config/server.xml for features and includes.
2022-01-20T17:02:51.4405455Z [INFO] [INFO] Generated the following features: [servlet-4.0]
2022-01-20T17:02:51.4406642Z [INFO] [INFO] Running liberty:deploy
2022-01-20T17:02:51.4407306Z [INFO] [INFO] CWWKM2102I: Using artifact based assembly archive : io.openliberty:openliberty-kernel:null:21.0.0.9:zip.
2022-01-20T17:02:51.4408076Z [INFO] [INFO] CWWKM2102I: Using installDirectory : /tmp/temp3087577316882393089/target/liberty/wlp.
2022-01-20T17:02:51.4408576Z [INFO] [INFO] CWWKM2102I: Using serverName : defaultServer.
2022-01-20T17:02:51.4409562Z [INFO] [INFO] CWWKM2102I: Using serverDirectory : /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer.
2022-01-20T17:02:51.4410153Z [INFO] [INFO] Copying 2 files to /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer
2022-01-20T17:02:51.4410927Z [INFO] [INFO] CWWKM2144I: Update server configuration file server.xml from /tmp/temp3087577316882393089/src/main/liberty/config/server.xml.
2022-01-20T17:02:51.4411537Z [INFO] [INFO] CWWKM2144I: Update server configuration file bootstrap.properties from inlined configuration.
2022-01-20T17:02:51.4412691Z [INFO] [INFO] CWWKM2185I: The liberty-maven-plugin configuration parameter "appsDirectory" value defaults to "dropins".
2022-01-20T17:02:51.4413391Z [INFO] [INFO] CWWKM2160I: Installing application dev-sample-proj-1.0-SNAPSHOT.war.xml.
2022-01-20T17:02:51.4414552Z [INFO] [INFO] CWWKM2010I: Searching for CWWKZ0001I.*dev-sample-proj-1.0-SNAPSHOT in /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/logs/messages.log. This search will timeout after 40 seconds.
2022-01-20T17:02:51.4424417Z [INFO] [INFO] CWWKM2015I: Match number: 1 is {"type":"liberty_message","host":"fv-az32-87.xh5foii4tneu1oj5xw13sxakeb.cx.internal.cloudapp.net","ibm_userDir":"\/tmp\/temp3087577316882393089\/target\/liberty\/wlp\/usr\/","ibm_serverName":"defaultServer","message":"CWWKZ0001I: Application dev-sample-proj-1.0-SNAPSHOT started in 0.974 seconds.","ibm_threadId":"00000021","ibm_datetime":"2022-01-20T17:01:57.398+0000","ibm_messageId":"CWWKZ0001I","module":"com.ibm.ws.app.manager.AppMessageHelper","loglevel":"AUDIT","ibm_sequence":"1642698117398_000000000000F"}.
2022-01-20T17:02:51.4425587Z [INFO] [INFO] Running liberty:install-feature
2022-01-20T17:02:51.4426490Z [INFO] [INFO] CWWKM2102I: Using artifact based assembly archive : io.openliberty:openliberty-kernel:null:21.0.0.9:zip.
2022-01-20T17:02:51.4427252Z [INFO] [INFO] CWWKM2102I: Using installDirectory : /tmp/temp3087577316882393089/target/liberty/wlp.
2022-01-20T17:02:51.4429101Z [INFO] [INFO] CWWKM2102I: Using serverName : defaultServer.
2022-01-20T17:02:51.4434243Z [INFO] [INFO] CWWKM2102I: Using serverDirectory : /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer.
2022-01-20T17:02:51.4436281Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/server.xml for features and includes.
2022-01-20T17:02:51.4444962Z [INFO] [INFO] Parsing the server file /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml for features and includes.
2022-01-20T17:02:51.4445892Z [INFO] [INFO] Installing features: [servlet-4.0, mpRestClient-1.2, cdi-2.0, mpHealth-1.0, jsonp-1.1, mpConfig-1.3, jaxrs-2.1]
2022-01-20T17:02:51.4446619Z [INFO] 
2022-01-20T17:02:51.4447057Z [INFO] [INFO] [AUDIT   ] CWWKG0016I: Starting server configuration update.
2022-01-20T17:02:51.4450613Z [INFO] [INFO] [AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml
2022-01-20T17:02:51.4451351Z [INFO] [INFO] [AUDIT   ] CWWKG0018I: The server configuration was not updated. No functional changes were detected.
2022-01-20T17:02:51.4454582Z [INFO] [INFO] [AUDIT   ] CWWKT0017I: Web application removed (default_host): http://fv-az32-87.xh5foii4tneu1oj5xw13sxakeb.cx.internal.cloudapp.net:9080/dev-sample-proj-1.0-SNAPSHOT/
2022-01-20T17:02:51.4455479Z [INFO] [INFO] [AUDIT   ] CWWKZ0009I: The application dev-sample-proj-1.0-SNAPSHOT has stopped successfully.
2022-01-20T17:02:51.4458219Z [INFO] [INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://fv-az32-87.xh5foii4tneu1oj5xw13sxakeb.cx.internal.cloudapp.net:9080/dev-sample-proj-1.0-SNAPSHOT/
2022-01-20T17:02:51.4459089Z [INFO] [INFO] [AUDIT   ] CWWKZ0003I: The application dev-sample-proj-1.0-SNAPSHOT updated in 0.351 seconds.
2022-01-20T17:02:51.4459850Z [INFO] [INFO] Product validation completed successfully.
2022-01-20T17:02:51.4460746Z [INFO] [INFO] The following features have been installed: cdi-1.2 mpHealth-1.0 servlet-3.1 json-1.0 
2022-01-20T17:02:51.4461490Z [INFO] [INFO] Source compilation was successful.
2022-01-20T17:02:51.4464784Z [INFO] [INFO] Copied file: /tmp/temp3087577316882393089/src/main/liberty/config/configDropins/overrides/generated-features.xml to: /tmp/tempConfig2736735446901469460/configDropins/overrides/generated-features.xml
2022-01-20T17:02:51.4465504Z [INFO] [INFO] Parsing the server file /tmp/tempConfig2736735446901469460/server.xml for features and includes.
2022-01-20T17:02:51.4467783Z [INFO] [INFO] Parsing the server file /tmp/tempConfig2736735446901469460/configDropins/overrides/generated-features.xml for features and includes.
2022-01-20T17:02:51.4471087Z [INFO] [INFO] Copied file: /tmp/temp3087577316882393089/src/main/liberty/config/configDropins/overrides/generated-features.xml to: /tmp/temp3087577316882393089/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml
2022-01-20T17:02:51.4472469Z [INFO] [INFO] Running maven-compiler-plugin:testCompile on /tmp/temp3087577316882393089/pom.xml
2022-01-20T17:02:51.4473092Z [INFO] [INFO] Changes detected - recompiling the module!
2022-01-20T17:02:51.4473973Z [INFO] [INFO] Compiling 1 source file to /tmp/temp3087577316882393089/target/test-classes
2022-01-20T17:02:51.4474574Z [INFO] [INFO] Running maven-resources-plugin:testResources
2022-01-20T17:02:51.4475372Z [INFO] [INFO] Using 'UTF-8' encoding to copy filtered resources.
2022-01-20T17:02:51.4511381Z [INFO] [INFO] skip non existing resourceDirectory /tmp/temp3087577316882393089/src/test/resources
2022-01-20T17:02:51.4515621Z [INFO] ====================== END ========================
2022-01-20T17:02:51.4516484Z [INFO] 
2022-01-20T17:02:51.4516880Z [INFO] 	at org.junit.Assert.fail(Assert.java:88)
2022-01-20T17:02:51.4517660Z [INFO] 	at org.junit.Assert.assertTrue(Assert.java:41)
2022-01-20T17:02:51.4518194Z [INFO] 	at net.wasdev.wlp.test.dev.it.DevTest.testDirectoryTest(DevTest.java:121)
2022-01-20T17:02:51.4519142Z [INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-01-20T17:02:51.4519800Z [INFO] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2022-01-20T17:02:51.4520862Z [INFO] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-01-20T17:02:51.4521472Z [INFO] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2022-01-20T17:02:51.4522359Z [INFO] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
2022-01-20T17:02:51.4522981Z [INFO] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2022-01-20T17:02:51.4523954Z [INFO] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
2022-01-20T17:02:51.4524618Z [INFO] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2022-01-20T17:02:51.4525596Z [INFO] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
2022-01-20T17:02:51.4526172Z [INFO] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
2022-01-20T17:02:51.4527131Z [INFO] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
2022-01-20T17:02:51.4527694Z [INFO] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
2022-01-20T17:02:51.4528561Z [INFO] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
2022-01-20T17:02:51.4529099Z [INFO] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
2022-01-20T17:02:51.4529977Z [INFO] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
2022-01-20T17:02:51.4530491Z [INFO] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
2022-01-20T17:02:51.4531412Z [INFO] 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2022-01-20T17:02:51.4532035Z [INFO] 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2022-01-20T17:02:51.4533201Z [INFO] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
2022-01-20T17:02:51.4533841Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
2022-01-20T17:02:51.4534848Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
2022-01-20T17:02:51.4535536Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
2022-01-20T17:02:51.4536529Z [INFO] 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
2022-01-20T17:02:51.4537237Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
2022-01-20T17:02:51.4538326Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
2022-01-20T17:02:51.4539137Z [INFO] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
2022-01-20T17:02:51.4539980Z [INFO] 
```

--- a/liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java
+++ b/liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java
@@ -91,8 +91,6 @@ public class DevTest extends BaseDevTest {
       assertTrue(propertiesFile.delete());
       assertTrue(verifyFileDoesNotExist(targetPropertiesFile, 5000));
    }
-   
-   @Ignore // TODO enable when https://github.com/OpenLiberty/ci.maven/issues/1383 is fixed
    @Test
    public void testDirectoryTest() throws Exception {
       // create the test directory
@@ -112,6 +110,8 @@ public class DevTest extends BaseDevTest {
       assertTrue(getLogTail(), verifyFileExists(unitTestTargetFile, 6000));
       long lastModified = unitTestTargetFile.lastModified();
 
+      // The resolution of File.lastModified() is 1000 ms so wait long enough for lastModified() to register the modification.
+      Thread.sleep(2000);
       // modify the test file
       String str = "// testing";
       BufferedWriter javaWriter = new BufferedWriter(new FileWriter(unitTestSrcFile, true));

